pkg/verify/verify_test.go

```diff
@@ -162,206 +160,19 @@ func Test_releaseVerifier_Verify(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			v := &releaseVerifier{
+			v := &ReleaseVerifier{
```

Comment:
`NewReleaseVerifier`?

Reply:
No, I want to set signature cache explicitly.